### PR TITLE
Update for v0.8

### PR DESCRIPTION
**Pull request by kastman**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by kastman** [comment]
@conda-forge-admin, please rerender

**Comment by kastman** [comment]
@conda-forge-admin, please restart ci

**Comment by kastman** [comment]
@conda-forge-admin, please rerender

**Comment by kastman** [comment]
@conda-forge-admin, please update for conda-build 3

**Comment by kastman** [comment]
@conda-forge-admin, please rerender

**Comment by kastman** [comment]
@conda-forge/bot  Something odd is going on with this feedstock; to begin with the autotick bot hasn't noticed a new version on pypi. 

Additionally when I manually tried to fix it, I somehow I lost all the various builds when conda-smithy re-rendered my feedstock, and the current version doesn't seem to go along with the doc (I didn't have a "providers" key in `conda-forge.yaml`, for example). Is someone available who could take a quick look, or tell me which team to ping instead? Thanks,

**Comment by beckermr** [comment]
This feedstock is not currently built on python3 and python 2.7 has been deprecated. You can use the python 2.7 admin command to add python 2.7 back

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
Looks like the URL for the source tarball is wrong @kastman. Fix that and update the SHA256 and you should be good to go!

**Comment by kastman** [comment]
@beckermr Thanks; I had missed that py3 was skipped, and didn't realize that py2 is now off by default. Much appreciated!

**Comment by kastman** [comment]
@beckermr linux and osx builds were started; I believe in the past win was also built and I don't see any os-related skips in the `recipe/meta.yaml`, and I do have `provider: win: azure` in the `conda-forge.yml`. Am I missing another default to prevent building windows? Thanks,

**Comment by beckermr** [comment]
We cannot build py 2.7 on win any more.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,6 @@ source:
 
 build:
   number: 0
-  skip: true  # [py3k]
```
Comment (beckermr) [inline recipe/meta.yaml]: You need to also remove the conda_build_config.yaml to get py3 builds.

Comment (kastman) [reply recipe/meta.yaml]: Besides enabling py2, is there any reason to keep the conda_build_config.yaml?

**Comment by kastman** [comment]
@conda-forge-admin rerender

**Comment by kastman** [comment]
@beckermr Sorry I'm really messing things up. I removed conda_build_config.yaml completely as you recommended and the CI built py2 packages, so I re-rendered and it looks like I lost everything. Should I add conda_build_config.yaml back in and add `python: 3`?

**Comment by kastman** [comment]
> @beckermr Sorry I'm really messing things up. I removed conda_build_config.yaml completely as you recommended and the CI built py2 packages, so I re-rendered and it looks like I lost everything. Should I add conda_build_config.yaml back in and add `python: 3`?

Sorry, just jumping the gun. CI picked up py36 and py37 afterall. Nevermind!

**Comment by kastman** [comment]
@beckermr thanks for the help!

@arokem I think this is good to merge now.

**Comment by kastman** [comment]
@arokem Happy to add myself to the maintainers list; think this is ready to merge.

File: recipe/meta.yaml (hunk 1 above)
Comment (beckermr) [reply recipe/meta.yaml:14]: Nope.

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@kastman I will merge once the ci passes

**Pull request merged**